task/sanity-inspect-image/0.1/README.md

```diff
@@ -0,0 +1,23 @@
+# sanity-inspect-image task
+
+## Description:
+Use Skopeo to inspect and analyze manifest data from the target source image of a container.
+
+## Params:
+
+| name         | description                                 |
+|--------------|---------------------------------------------|
+| IMAGE_URL    | The fully qualified image name.             |
+| IMAGE_DIGEST | Image digest.                               |
+| DOCKER_AUTH  | Secret with config.json for container auth. |
+
+## Results:
+
+| name                  | description                                |
+|-----------------------|--------------------------------------------|
+| BASE_IMAGE            | Base image the source image is built from. |
+| BASE_IMAGE_REPOSITORY | Base image repository URL.                 |
+| HACBS_TEST_OUTPUT     | Test output.                               |
+
```

Comment:
I think it might be good to also add a link to the source repository, something like this:
```
## Source repository for image:
https://github.com/redhat-appstudio/hacbs-test
```